src/components/Help/help.md

```diff
@@ -0,0 +1,43 @@
+# Jaeger UI Help
+
+
+## Work In Progress
+
+This document is a work in progress. To request additional documentation of the Jaeger UI, please create a [documentation ticket](https://github.com/jaegertracing/jaeger-ui/issues/new?labels=documentation).
+
+
+## Keyboard Shortcuts
```

Comment:
I think this should go in a modal for the Trace page, since its specific to that. A little tooltip can open it up